modules/Bio/EnsEMBL/Variation/Pipeline/SpliceAI/SplitFiles.pm

```diff
@@ -0,0 +1,235 @@
+  }
+
+  # prepare input vcf file for tabix
+  my ($exit_code, $stderr, $flat_cmd) = $self->run_system_command("bgzip $vcf_file_path");
+  my ($exit_code, $stderr, $flat_cmd) = $self->run_system_command("tabix -p vcf $vcf_file_path.gz");
+
+  # Create directory to store the input file only with the variants of interest (overlapping new transcripts)
+  my $vcf_file_path_subset = $main_dir . '/input_vcf_files_subset';
+  if (! -e $vcf_file_path_subset) {
+    $self->create_dir($vcf_file_path_subset);
+  }
+
+  $self->param('input_dir_subset', $vcf_file_path_subset);
+
+  open(my $write, '>', $vcf_file_path_subset . '/' . $vcf_file) or die $!;
+
+  my $positions_of_interest = $transcripts->{$chr};
+  foreach my $position (@$positions_of_interest) {
+    my @positions = split /-/, $position;
+    my $transcript_start = $positions[0];
+    my $transcript_end = $positions[1];
+
+    my $pos_string = sprintf("%s:%i-%i", $chr, $transcript_start, $transcript_end);
+
+    (open my $read, "tabix  " . $input_dir . "/" . $vcf_file . ".gz $pos_string |")
+      || die "Failed to read from input vcf file " . $input_dir . "/" . $vcf_file . ".gz \n" ;
+
+    while (my $row = <$read>) {
+      chomp $row;
+      next if($row =~ /^#/);
+
+      my @columns = split /\t/, $row;
+      my @alt_splitted = split /,/, $columns[4];
+      foreach my $x (@alt_splitted) {
+        print $write $columns[0] . "\t" . $columns[1] . "\t" . $columns[2] . "\t" . $columns[3] . "\t" . $x . "\t.\t.\t.\n";
+      }
+    }
+    close($read);
+  }
+  close($write);
+
+  # Sort new vcf file
+  my $vcf_file_subset = $vcf_file_path_subset . '/' . $vcf_file;
+  my $vcf_file_subset_sorted = $vcf_file_path_subset . '/sorted_' . $vcf_file;
+  my ($exit_code, $stderr, $flat_cmd) = $self->run_system_command("sort -t \$'\t' -k1,1 -k2,2n $vcf_file_subset > $vcf_file_subset_sorted");
+  ($exit_code, $stderr, $flat_cmd) = $self->run_system_command("mv $vcf_file_subset_sorted $vcf_file_subset");
+}
+
+# Check if there are new MANE transcripts since last release
+sub get_new_transcripts {
+  my $self = shift;
+
+  my %new_transcripts;
+
+  my $registry = 'Bio::EnsEMBL::Registry';
+  my $registry_file = $self->param('registry');
+  $registry->load_all($registry_file);
+  # Only for human
+  my $cdba =  $registry->get_DBAdaptor('Homo_sapiens', 'core');
+  my $dbh = $cdba->dbc->db_handle;
+
+  # Get new transcripts which need to have the scores re-calculated
+  my $sth = $dbh->prepare(qq{ SELECT s.name,t.stable_id,t.version,g.seq_region_start,g.seq_region_end FROM transcript t
+                              JOIN transcript_attrib ta ON t.transcript_id = ta.transcript_id
+                              JOIN seq_region s ON t.seq_region_id = s.seq_region_id
+                              JOIN gene g ON g.gene_id = t.gene_id
+                              WHERE t.stable_id like 'ENST%' and t.biotype = 'protein_coding' and ta.attrib_type_id = 535 and t.modified_date >= DATE_SUB(NOW(), INTERVAL 4 MONTH) });
```

Comment:
I would suggest to use the attrib value. The id shouldn't change. But it is a lot less likely that the attrib name changes. Using the attrib value also makes it more clear by what you are filtering.